src/components/dialogs/parameters/parameters.tsx

```diff
@@ -581,27 +581,18 @@ export function useParameterState(paramName: UseParameterStateParamName) {
         setParamLocalState(paramGlobalState);
     }, [paramGlobalState]);
 
-    const backendupdateConfigParameterCb = useCallback(
-        (paramName: string, newParams: any) => {
-            updateConfigParameter(paramName, newParams).catch((error) => {
+    const handleChangeParamLocalState = useCallback(
+        (value: any) => {
+            setParamLocalState(value);
+            updateConfigParameter(paramName, value).catch((error) => {
                 setParamLocalState(paramGlobalState);
                 snackError({
                     messageTxt: error.message,
                     headerId: 'paramsChangingError',
                 });
             });
         },
-        [paramGlobalState, snackError]
-    );
-
-    const debouncedBackendupdateConfigParameterCb = useDebounce(backendupdateConfigParameterCb, 1000);
-
-    const handleChangeParamLocalState = useCallback(
-        (value: any) => {
-            setParamLocalState(value);
-            debouncedBackendupdateConfigParameterCb(paramName, value);
-        },
-        [debouncedBackendupdateConfigParameterCb, paramName]
+        [paramName, snackError, setParamLocalState, paramGlobalState]
```

Comment:
a `setState` is useless in a useCallback dependency array

remove `setParamLocalState`

It would be appreciated to have thoses changes in a separeted PR

Reply:
Ok, done here : https://github.com/gridsuite/gridstudy-app/pull/2559